client/packages/glsp-sprotty/src/utils/smodel-util.ts

```diff
@@ -42,7 +51,7 @@ export function isSelected(element: SModelElement): element is SModelElement & S
 }
 
 export function isSelectedBoundsAware(element: SModelElement): element is SModelElement & BoundsAware & Selectable {
-    return isBoundsAware(element) && isSelected(element);
+    return isBoundsAware(element) && isSelected(element) && !isRoutable(element);
```

Comment:
I am not sure why whether it is a good idea to simply add this here since the function name does not reflect this. There only seems to be a single place where this is currently used (change-bounds-tool) so we could simply have added the condition there as well or renamed the function here.

Reply:
Good point. For now, it should suffice to add the condition to the change-bounds-tool